app/interactors/show_group.rb

```diff
@@ -1,3 +1,9 @@
-class ShowGroup
-  include Interactor
+class ShowGroup < StandardInteraction
+  def validate_input
+    context.fail!(errors: "invalid input") unless context.id
+  end
+
+  def execute
+    context.group = Group.find(context.id)
+  end
```

Comment:
I've found myself either rescuing the error so I can just return one of my standard error objects or, in this case, using `find_by` so that it doesn't throw an error and just returns `nil` if nothing was found. Then I could validate output and proceed as usual.